README.md

```diff
@@ -107,3 +107,14 @@ TBD:
 * when using separate celery app, send SIGUSR1 from master to workers to trigger
     soft shutdown handling
     (see `video_transcoding.celery.send_term_to_children`)
+
+### Settings
+
+Application settings can be set up via env variables, see `video_transcoding.defaults`.
+Also defaults can be overridden this via `django.conf.settings.VIDEO_TRANSCODING_CONFIG`.
+
+### Model inheritance
+
+For preset-related models use `<Model>Base` abstract models defined in `video_transcoding.models`.
+For overriding `Video` model set `VIDEO_TRANSCODING_CONFIG["VIDEO_MODEL"]` key to `app_label.ModelName` in `settings`.
+Connect other django models to `Video` using `video_transcoding.models.get_video_model()`. 
```

Comment:
Will it work correctly with migrations? Same as User model?

Reply:
There are guards like `if not Video._meta.abstract` in migrations, so yes, this will work for both cases.
When you use default models, migrations from library do the job. 
When you replace video model with your own, you manage migrations yourself in your project.

Reply:
At least in test project (default models) and in cms (abstract + overridden) migrations created and implemented correctly. It's tricky to jump over in the fly, but it's also possible.